test-suite.html

```diff
@@ -27,12 +28,11 @@ <h1>Running the test suite</h1>
 	<p>Running the test suite is simple: just call <code class="language-bash">npm test</code>.</p>
 	<p>All test files are run in isolation. A new prism instance is created for each test case. This will slow the test runner a bit down, but we can be sure that nothing leaks into the next test case.</p>
 
-	<section id="running-tests-for-specific-languages">
-		<h2>Running tests for specific languages</h2>
 
-		<p>To run the tests only for one language, you can use the <code>language</code> parameter: <code class="language-bash">npm run test:languages -- --language=markup</code>.</p>
-		<p>You can even specify multiple languages: <code class="language-bash">npm run test:languages -- --language=markup --language=css</code>.</p>
-	</section>
+	<h2 id="running-tests-for-specific-languages">Running tests for specific languages</h2>
+
+	<p>To run the tests only for one language, you can use the <code>language</code> parameter: <code class="language-bash">npm run test:languages -- --language=markup</code>.</p>
```

Comment:
Force command to a new line to avoid wrapping (observed in Chrome on 1080p display):

![image](https://user-images.githubusercontent.com/7819991/64950632-2207ed80-d87c-11e9-86e9-67fb1a41ee31.png)


```suggestion
	<p>To run the tests only for one language, you can use the <code>language</code> parameter:<br><code class="language-bash">npm run test:languages -- --language=markup</code>.</p>
```